sherpa/plot/backends.py

```diff
@@ -163,8 +256,9 @@ def end(self):
         '''Called from the UI after an interactivep lot is done'''
         pass
 
+    @translate_args
     def plot(self, x, y, *,
```

Comment:
Ohh. I hadn't noticed this in the earlier commit. I think this may be our first use of keyword-only arguments. I have had a number of commits where I've changed `foo(a, b, c, d, e, ...)` to ones where the arguments are explicitly named (as I have seen bugs in sherpa from things in the wrong order), so +1 here.